tests/suite/test_rl_policies.py

```diff
@@ -26,14 +26,18 @@
 )
 
 
-@pytest.mark.policies
+@pytest.mark.test
```

Comment:
The test marker of `policies` made sense to me, can you explain this change?

Reply:
Thanks for catching that, was a local change to test individual class which got pushed.